HomePage.js

```diff
@@ -133,21 +115,11 @@ var HomePage = React.createClass({
 
   renderNewAlbum: function(album) {
     return(
-      <TouchableHighlight
```

Comment:
The main change here is that the Homepage, shouldn't really know how to render a row. The row object should know how to render a row - so what we have is better separation of concerns... and smaller files (yay!)